dist/json-refs.cjs.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+// this file might look strange and you might be wondering what it's for
+// it's lets you import your source files by importing this entrypoint
+// as you would import it if it was built with preconstruct build
+// this file is slightly different to some others though
+// it has a require hook which compiles your code with Babel
+// this means that you don't have to set up @babel/register or anything like that
+// but you can still require this module and it'll be compiled
+
+// this bit of code imports the require hook and registers it
+let unregister = require("../node_modules/@preconstruct/hook").___internalHook(typeof __dirname === 'undefined' ? undefined : __dirname, "..", "..");
+
+// this re-exports the source file
+module.exports = require("../src/index.js");
+
+unregister();
```

Comment:
Why is this required?  I read the comment but I'm not sure why Node.js would have an issue loading Node.js sources.

Reply:
This is a `preconstruct` output - the `src/index.js` is now an ESM module (import/export), so this shim is pull in @babel/register when working with the module locally.

Reply:
The preconstruct build here was using the dev build instead of the production one - I've updated the committed file, but as suggested below, I think we should stop committing the dist/ folder.